configs/components/module-puppetlabs-augeas_core.json

```diff
@@ -0,0 +1 @@
+{"url":"git@github.com:puppetlabs/puppetlabs-augeas_core.git","ref":"refs/tags/1.0.0"}
```

Comment:
Other components tend to use the read-only `git://` protocol -- I don't really see a strong reason _not_ to use `git@` here, though.

Reply:
ah, my bad. The repo was private, let me fix that